### ISSUE TITLE
R package for access to the Depsy API?

### ISSUE THREAD

**Issue by maelle** [ropensci/unconf16#7]
Depsy API: http://depsy.org/api/ "Depsy helps build the software-intensive science of the future by promoting credit for software as a fundamental building block of science."

Maybe having a R package for accessing the API would be interesting? Or at least discussing whether Depsy is useful?

**Comment by leeper** [comment]
:+1: Definitely think it would be useful to discuss Depsy and the broader issue of attributing credit for contributions to research software.

**Comment by karthik** [comment]
+1
I've been working on a side project to make it easy for software developers to quickly publish a citable paper. We're doing a soft launch in two weeks and also ties in with Depsy so this would be a neat idea to discuss/hack on at the unconf.

**Comment by geanders** [comment]
+1
As a small side part of this, it would be useful to develop a standard for how to cite grant support in package metadata. Then a project officer could search Depsy by grant number to see how well-used packages developed under a grant are-- I don't think you can do that yet? It seems like it could be an easy add-on if we found a good role to use for funding support in the "authors" meta-data for a package.

**Comment by cboettig** [comment]
@geanders Definitely! That's something we're hoping to help address with CodeMeta, e.g. https://github.com/codemeta/codemeta/blob/master/crosswalk.csv. Having a standard way to declare this information would mean, for instance, that an author could then add a little `.json` file to a GitHub repo, and [import the repo to zenodo for a DOI](https://guides.github.com/activities/citable-code/), and presto, the grant number would be part of the Zenodo metadata which is associated with that DOI and added to the central DataCite metadata.  Program officers or anyone else could then search the CrossRef/DataCite data and see any product that has a DOI and comes from a given grant.  

Exactly how to implement this in a way that is easy for any software project and also easy to search is an open discussion.  Would love to hear your and other thoughts on this.
